ledger/src/blockstore.rs

```diff
@@ -3441,6 +3457,30 @@ fn handle_chaining(
     Ok(())
 }
 
+/// A helper function of handle_chaining which handles the chaining based
+/// on the `SlotMetaWorkingSetEntry` of the specified `slot`.  Specifically,
+/// it handles the following two things:
+///
+/// First, based on the `SlotMetaWorkingSetEntry` of the input `slot`, it it
```

Comment:
nits:
- "of the input `slot`" ==> "for `slot`" (putting inside the backticks implies it is the input parameter)
- Extra "it" in this sentence, thinking the first one is supposed to be "if"?
- This first sentence is a tad long; you could maybe break it up into two (incorporating my previous comments)
    - "First, based on the `SlotMetaWorkingSetEntry` for `slot`, check if `slot` did not previously have a parent slot but does now."
    - "If `slot` satisfies this condition, update the Orphan property of both `slot` and its parent slot based on their current orphan status."